lib/api/objectGet.js

```diff
@@ -117,6 +119,14 @@ function objectGet(authInfo, request, returnTagCount, log, callback) {
               dataLocator[0].dataStoreType === 'azure') {
                 dataLocator[0].azureStreamingOptions = streamingParams;
             }
+            if (dataLocator && dataLocator[0] &&
```

Comment:
@dora-korpar @nicolas2bert is it possible object stored in aws backend will have multiple items in the dataLocator array? :S

Reply:
@electrachong no, only one: `dataLocator[0]`

Reply:
👍 thanks!!

Reply:
why are we doing this for just aws and why in the objectGet api versus in the aws client itself? I thought the point was to prevent us from starting to stream data and then dropping the connection once we realize there is no data. It seems it would be cleaner to put this in the get calls of the azure and aws clients.